megalit.py

```diff
@@ -1,5 +1,196 @@
-from game import Game
+# this file contains the Megalit class and its Slab helper class
 
+import builtins
```

Comment:
What is this import needed for? Googling makes it seem like it's for Python 2/3 compatibility

Reply:
i think vs code added it, but it also seems to be unused currently so i removed it